Sources/VergeStore/Store/DispatcherType.swift

```diff
@@ -77,7 +77,7 @@ extension DispatcherType {
     _ file: StaticString = #file,
     _ function: StaticString = #function,
     _ line: UInt = #line,
-    mutation: (inout Scope) throws -> Result
+    mutation: (inout UnsafeInoutReference<Scope>) throws -> Result
```

Comment:
The reason why we still use `inout` attribute is to restrict escaping the reference object by developers in compile time.